test/documentation-content/test-toolkit/1.0/antora.yml

```diff
@@ -0,0 +1,14 @@
+name: test-toolkit
+title: Bonita test toolkit
+version: 1.0
+prerelease: beta
+start_page: ROOT:process-testing-overview.adoc
+asciidoc:
+  attributes:
+    # page attributes
+    page-editable: true
+    page-out-of-support: false
+    page-pagination: true # display links to previous/next pages at the bottom of the page
```

Comment:
❓ we could add the `page-custom-message` attribute here for demonstration
See https://github.com/bonitasoft/bonita-test-toolkit-doc/commit/54bdc5eeb909b95aa3d6ed9f9a75fbe2587e6361#diff-8d4aad078b0eff4f312f62348b44d8f4b91666db46b69337e60751fb700068f9R13